CMake/External_VTK.cmake

```diff
@@ -54,13 +54,31 @@ set(vtk_cmake_args ${vtk_cmake_args}
   )
 
 # Qt
-add_package_dependency(
-  PACKAGE VTK
-  PACKAGE_DEPENDENCY Qt
-  PACKAGE_DEPENDENCY_ALIAS Qt4
-)
-if(QT_QMAKE_EXECUTABLE)
-  set(BUILD_QT_WEBKIT ${QT_QTWEBKIT_FOUND})
+if(Qt_version_major EQUAL 4)
```

Comment:
Question: What happens if fletch Qt is disabled? Can fletch VTK be compiled against external Qt?

Reply:
Yes. (In fact, that's what I tested — for Qt5, anyway — since I don't want to wait on Qt to build :stuck_out_tongue_closed_eyes:.)

Reply:
That's awesome. Thanks 👍